x-pack/plugins/stack_connectors/server/connector_types/index.ts

```diff
@@ -64,26 +63,24 @@ export {
 
 export function registerConnectorTypes({
   actions,
-  logger,
   publicBaseUrl,
 }: {
   actions: ActionsPluginSetupContract;
-  logger: Logger;
   publicBaseUrl?: string;
 }) {
-  actions.registerType(getEmailConnectorType({ logger, publicBaseUrl }));
-  actions.registerType(getIndexConnectorType({ logger }));
-  actions.registerType(getPagerDutyConnectorType({ logger }));
-  actions.registerType(getSwimlaneConnectorType({ logger }));
-  actions.registerType(getServerLogConnectorType({ logger }));
-  actions.registerType(getSlackConnectorType({ logger }));
-  actions.registerType(getWebhookConnectorType({ logger }));
-  actions.registerType(getCasesWebhookConnectorType({ logger }));
-  actions.registerType(getXmattersConnectorType({ logger }));
-  actions.registerType(getServiceNowITSMConnectorType({ logger }));
-  actions.registerType(getServiceNowSIRConnectorType({ logger }));
-  actions.registerType(getServiceNowITOMConnectorType({ logger }));
-  actions.registerType(getJiraConnectorType({ logger }));
-  actions.registerType(getResilientConnectorType({ logger }));
-  actions.registerType(getTeamsConnectorType({ logger }));
+  actions.registerType(getEmailConnectorType({ publicBaseUrl }));
+  actions.registerType(getIndexConnectorType());
+  actions.registerType(getPagerDutyConnectorType());
+  actions.registerType(getSwimlaneConnectorType());
+  actions.registerType(getServerLogConnectorType());
+  actions.registerType(getSlackConnectorType({}));
+  actions.registerType(getWebhookConnectorType());
+  actions.registerType(getCasesWebhookConnectorType());
+  actions.registerType(getXmattersConnectorType());
+  actions.registerType(getServiceNowITSMConnectorType());
+  actions.registerType(getServiceNowSIRConnectorType());
+  actions.registerType(getServiceNowITOMConnectorType());
```

Comment:
I am getting errors here when i try to run this PR locally.
I think getServiceNowSIRConnectorType and getServiceNowITOMConnectorType are still expecting the logger as param

Reply:
Thanks I will fix that.

Reply:
Should be resolved here 6572520ddf1e1546735d0d2ac152a4b234a02481